slick/src/main/scala/slick/jdbc/DerbyProfile.scala

```diff
@@ -89,6 +97,8 @@ trait DerbyProfile extends JdbcProfile {
     - JdbcCapabilities.booleanMetaData
     - JdbcCapabilities.supportsByte
     - RelationalCapabilities.repeat
+    - RelationalCapabilities.createIfNotExists
+    - RelationalCapabilities.dropIfExists
```

Comment:
does removing the capabilities here make sense? Looks like you added support below, so now the driver actually has this capability. No?